From b97e2bd1db22d9300add5dc4f16b75c14dba65e5 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 24 Jul 2022 08:50:28 +0000
Subject: [PATCH 71/75] WIP: dt-bindings: arm: amlogic: add Beelink GT1 binding

Add the board binding for the Shenzen AZW (Beelink) GT1 Android
Set-Top Box device.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index a17daa64ba31..8f7908bb3fee 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -132,6 +132,7 @@ properties:
           - enum:
               - amlogic,q200
               - amlogic,q201
+              - azw,gt1
               - azw,gt1-ultimate
               - khadas,vim2
               - kingnovel,r-box-pro
-- 
2.17.1

